### DISCUSSION TITLE
unable to locate pkg

### DISCUSSION THREAD

**Discussion by joker3597** [termux/termux-packages#15232] [Q&A] [answered]
im new here i always facing with that problem  i so many times tried to upgrade and update  i cant even upgrade and update same problem tried to change repo nothing change still same also reinstall termux but nothing change again pls help me

**Comment by sylirre** [comment, marked as answer]
Don't use Termux from Play Store.

The official up-to-date releases are at https://f-droid.org/ru/packages/com.termux/.
